commit cc49ad8dac24309f5452d5dea9abd406615a52d9
Author: Jordan Brown <github@jordan.maileater.net>
Date:   Sun Jul 18 21:01:46 2021 -0700
Upstream: merged
Subject: Upstream patch to fix build with cgal-5.3.

    Fix build failure with "generic_print_polyhedron" on CGAL-5.3.

Index: openscad-2021.01/src/cgalutils-polyhedron.cc
===================================================================
--- openscad-2021.01.orig/src/cgalutils-polyhedron.cc
+++ openscad-2021.01/src/cgalutils-polyhedron.cc
@@ -337,19 +337,6 @@ namespace CGALUtils {
     }
 	};
 
-	template <typename Polyhedron>
-	std::string printPolyhedron(const Polyhedron &p) {
-		std::ostringstream sstream;
-		sstream.precision(20);
-
-    Polyhedron_writer writer;
-    generic_print_polyhedron(sstream, p, writer);
-		
-		return sstream.str();
-	}
-
-	template std::string printPolyhedron(const CGAL_Polyhedron &p);
-
 }; // namespace CGALUtils
 
 #endif /* ENABLE_CGAL */
Index: openscad-2021.01/src/cgalutils.h
===================================================================
--- openscad-2021.01.orig/src/cgalutils.h
+++ openscad-2021.01/src/cgalutils.h
@@ -45,7 +45,6 @@ namespace CGALUtils {
 	bool is_approximately_convex(const PolySet &ps);
 	Geometry const* applyMinkowski(const Geometry::Geometries &children);
 
-	template <typename Polyhedron> std::string printPolyhedron(const Polyhedron &p);
 	template <typename Polyhedron> bool createPolySetFromPolyhedron(const Polyhedron &p, PolySet &ps);
 	template <typename Polyhedron> bool createPolyhedronFromPolySet(const PolySet &ps, Polyhedron &p);
 	template <class Polyhedron_A, class Polyhedron_B> 
